data/FilterListTag.json

```diff
@@ -1499,6 +1499,22 @@
     "filterListId": 493,
     "tagId": 11
   },
+  {
+    "filterListId": 500,
```

Comment:
just removed 500-503 in https://github.com/collinbarrett/FilterLists/commit/2793669b784d66014b829e86eb302938453bc36e since adzhosts is offline. sorry for stepping on your toes, but please remove these from your PR as well. thanks.

Reply:
Should be fixed now.

Reply:
yep, thanks!